src/client/components/login/login.stories.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import Login from './login.component';
+import { action } from '@storybook/addon-actions';
+import { withKnobs, text, select } from '@storybook/addon-knobs';
+
+export default {
+  title: 'ContainerLayouts/Login',
+  component: Login,
+  decorators: [withKnobs],
+};
+
+const notifications = {
+  None: null,
+  Email: 'email',
+  Password: 'password',
+  General: 'general',
+};
+
+export const LoginContainer = () => {
```

Comment:
Since it's not a container component let's not call the story that.

I realize that you import the Login component under the name Login, so you can't use that, which I find really annoying with the new way of doing storybook, but how about:

```suggestion
export const LoginForm = () => {
```